dist/husky.js

```diff
@@ -47619,9 +47620,30 @@ define('__component__$url-input@husky',['services/husky/url-validator'], functio
                         JustifyBlock: 'align-justify',
                         NumberedList: 'list-ol',
                         BulletedList: 'list',
-                        PasteFromWord: 'file-word-o',
+                        PasteFromWord: 'clipboard',
                         Styles: 'header',
                         Source: 'code'
+                    },
+                    relations = {
+                        Link: ['Unlink']
+                    },
+                    expandToolbar = function(toolbar) {
```

Comment:
Please provide a comment, on what the function does. The function is not that easy.